radiotools/coreas/generate_coreas_sim.py

```diff
@@ -169,67 +173,237 @@ def write_list(filename, station_positions, station_name=None, append=False):
     fout.close()
 
 
-def write_list_star_pattern(filename, zen, az, append=False, obs_level=1564.0, obs_level_corsika=None,
-                            inc=np.deg2rad(-35.7324), ground_plane=True, r_min=0., r_max=500.,
-                            rs=None,
-                            slicing_method=None, slices=[], n_rings=20,
-                            azimuths=np.deg2rad([0, 45, 90, 135, 180, 225, 270, 315]),
-                            gammacut=None):
+def write_list_star_pattern(filename, zenith, azimuth, 
+                            append=False, 
+                            obs_level=1400.0, 
+                            obs_level_corsika=None, 
+                            ground_plane=True,
+                            Auger_CS=True,
```

Comment:
`auger_cs` ?

Reply:
That was one of the main reason I started working on this. The old code implicitly assumed that you are using the Auger coordinate system for giving the input parameters and just rotated them to the system CORSIKA uses somewhere in the function. 
The main change I made is make this explicit and documented, and add the option to also use CORSIKA's coordinate system for the input. The Auger_CS option determines the input coordinate system.

Reply:
Sure, may comment was regarding all the capital letters in the variable name.

Reply:
good point, I'll make them lower case